apps/src/code-studio/pd/workshop_dashboard/attendance/session_attendance.jsx

```diff
@@ -49,6 +49,7 @@ const SessionAttendance = React.createClass({
     this.showSectionMembership = !this.shouldUseNewAttendance && this.props.accountRequiredForAttendance;
     this.showPuzzlesCompleted = this.shouldUseNewAttendance && this.isCSF;
     this.isAdmin = window.dashboard.workshop.permission === "workshop_admin";
+    this.isPlp = window.dashboard.workshop.permission.indexOf('plp') >= 0;
```

Comment:
Agreed. Note this is how we've done it other places too: https://github.com/code-dot-org/code-dot-org/search?utf8=%E2%9C%93&q=isPlp&type=

Perhaps this is a good time to wrap this up in a permissions object?